server/handlers/instrumented_project_command_output_handler.go

```diff
@@ -0,0 +1,44 @@
+package handlers
+
+import (
+	stats "github.com/lyft/gostats"
+	"github.com/runatlantis/atlantis/server/events/models"
+)
+
+type InstrumentedProjectCommandOutputHandler struct {
+	ProjectCommandOutputHandler
+	numChans stats.Gauge
+}
+
+func NewInstrumentedProjectCommandOutputHandler(prjCmdOutputHandler ProjectCommandOutputHandler, statsScope stats.Scope) ProjectCommandOutputHandler {
+	return &InstrumentedProjectCommandOutputHandler{
+		ProjectCommandOutputHandler: prjCmdOutputHandler,
+		numChans:                    statsScope.Scope("log_streaming").NewGauge("num_ws_chans"),
```

Comment:
I like to have an intuitive understanding of what a metric is doing usually by just looking at the name and if it doesn't make sense it might not be worth tracking.

In this case, I'm not sure what information I'm getting out of this.  num_ws_chans represents the number of websocket connections yeah?

If so, then I think we should name it appropriately.  Something like:

```
job.websocket.connections 
```

Also question on why you're using gauge here?